frontend/src/features/mapSideBar/layout/MapSideBarLayout.tsx

```diff
@@ -1,115 +1,108 @@
 import styled from 'styled-components';
 
 interface IMapSideBarLayoutProps {
-  show: boolean;
+  showSideBar: boolean;
   setShowSideBar: (show: boolean) => void;
   title: React.ReactNode;
-  hidePolicy?: boolean;
-  /** property name for title */
-  propertyName?: string;
-  header: React.ReactNode;
+  header?: React.ReactNode;
+  icon: React.ReactNode | React.FunctionComponent;
+  showCloseButton?: boolean;
 }
 
 /**
  * SideBar layout with control bar and then form content passed as child props.
  * @param param0
  */
 const MapSideBarLayout: React.FunctionComponent<IMapSideBarLayoutProps> = ({
-  show,
+  showSideBar,
   setShowSideBar,
-  hidePolicy,
   title,
-  propertyName,
+  header,
+  icon,
+  showCloseButton,
   ...props
 }) => {
   return (
-    <StyledMapSideBarLayout
-      className={classNames('map-side-drawer', {
-        show: show,
-      })}
-    >
+    <StyledMapSideBarLayout show={showSideBar}>
       <VisibilitySensor partialVisibility={true}>
         {({ isVisible }: any) => (
           <>
             <TitleBar>
-              <Underline>
-                <LotIcon className="mr-1" />
-                <Styled.H1 className="mr-auto">{title}</Styled.H1>
+              <Row>
+                <Col>
+                  <Styled.H1 className="mr-auto">
+                    {icon}
+                    {title}
+                  </Styled.H1>
+                </Col>
 
-                <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
-                  <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
-                </TooltipWrapper>
-              </Underline>
+                {showCloseButton && (
+                  <Col xs="auto">
+                    <TooltipWrapper toolTipId="close-sidebar-tooltip" toolTip="Close Form">
+                      <CloseIcon title="close" onClick={() => setShowSideBar(false)} />
+                    </TooltipWrapper>
+                  </Col>
+                )}
+              </Row>
+              <Underline />
             </TitleBar>
-            <Header>{props.header}</Header>
-            <Content>{isVisible ? props.children : null}</Content>
+
+            <StyledBody>
+              {header && isVisible && <Header>{header}</Header>}
+              <Content>{isVisible ? props.children : null}</Content>
+            </StyledBody>
           </>
         )}
       </VisibilitySensor>
     </StyledMapSideBarLayout>
   );
 };
 
-const Content = styled.div`
-  grid-area: content;
+const StyledBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
   width: 100%;
   height: 100%;
-  top: 2rem;
-  position: absolute;
-  margin-top: 1rem;
+  position: relative;
 `;
 
-const Header = styled.div`
-  grid-area: header;
+const Content = styled.div`
+  height: 100%;
+  width: 100%;
 `;
 
-const LotIcon = styled(LotSvg)`
-  width: 3rem;
-  height: 3rem;
-  align-self: flex-end;
-`;
+const Header = styled.div``;
```

Comment:
Is there a reason for these empty styled components?

Reply:
Leaving them here since I am not 100% we will need to revisit this as we work on the rest of the sidebar components